SekoiaIO/endpoint/_meta/fields.yml

```
@@ -6852,6 +6852,11 @@ action.properties.thumbPrint:
   name: action.properties.thumbPrint
   type: keyword
 
+action.properties.time_without_events:
+  description: Time without events in seconds
+  name: action.properties.time_without_events
+  type: integer
```

```suggestion
  type: long
```

`integer` is not officially supported by ingest